Hey @nrhodes this is great. Thank you very much for this work. The failing check is for python pep8 formatting. In particular:

```
tests/commands/grade_tests.py:398:30: E261 at least two spaces before inline comment
```

Could you get this fixed, and then I'd be very happy to push out a new version of `courseraprogramming`.

Thanks!

--- a/tests/commands/grade_tests.py
+++ b/tests/commands/grade_tests.py
@@ -395,7 +395,7 @@ def test_command_local_grade_simple(run_container, utils, common):
         "Id": "myContainerInstanceId",
     }
     utils.docker_client.return_value = docker_mock
-    h_config = {'foo': 'bar'} # just some unique value
+    h_config = {'foo': 'bar'}  # just some unique value
     docker_mock.create_host_config.return_value = h_config
 
     grade.command_grade_local(args)

